ReactNativeClient/lib/joplin-database.js

```diff
@@ -685,6 +685,48 @@ class JoplinDatabase extends Database {
 			if (targetVersion == 29) {
 				queries.push('ALTER TABLE version ADD COLUMN table_fields_version INT NOT NULL DEFAULT 0');
 			}
+			if (targetVersion == 30) {
+				queries.push('DROP TRIGGER IF EXISTS notes_fts_before_update');
+				queries.push('DROP TRIGGER IF EXISTS notes_fts_before_delete');
+				queries.push('DROP TRIGGER IF EXISTS notes_after_update');
+				queries.push('DROP TRIGGER IF EXISTS notes_after_insert');
+				queries.push('DROP TABLE IF EXISTS notes_fts');
+
+				const newVirtualTableSql = `
+					CREATE VIRTUAL TABLE notes_fts USING fts4(
+						content="notes",
+						notindexed="id",
```

Comment:
In `targetVersion` 18 we are not using `notes` table for the fts table content. Instead we are using another table called `notes_normalized`. Why do we need this table?

Reply:
This table is populated in SearchEngine::syncTable. It converts the note title and body to lowercase and removes diacritics.